### PR TITLE
Add a Contributing file

### PR DESCRIPTION
**Pull request by SamDepardieu**
Add a contribution file to guide future contributors through our values, our workflow and our rules.
At the moment, it is a copy from this file -> https://www.contributor-covenant.org/version/1/4/code-of-conduct
Pretty much everything is in it, but we should make sure everything we want lives in this file.
This file should be discussed to ensure that we all agree with what is in it.

### CODE REVIEW

**Comment by SamDepardieu** [comment]
[Create a contribution template for the parser](https://trello.com/c/vhhe1Cah/367-create-a-contribution-template-for-the-parser)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
[the remark this answers is not in the document]
Comment (SamDepardieu) [reply CONTRIBUTING.md]: I don't know actually 🤔 
I'm removing this bit, we'll come with something later.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+# Contributing
+
+:tada: First of all, welcome to this repository, and thanks for your interest, wether you want to contribute
```
Comment (nsorros) [inline CONTRIBUTING.md]: whether instead of wether

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+or simply are interested in opening an issue.
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
```
Comment (nsorros) [inline CONTRIBUTING.md]: via an issue instead of via issue

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+email, or any other method with the owners of this repository before making a change.
```
Comment (nsorros) [inline CONTRIBUTING.md]: or any other method? i assume we need to be specific about the method of communicating

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Before pushing your code, please make sure to have it tested and covered by some unit tests.
```
Comment (nsorros) [inline CONTRIBUTING.md]: tests need to pass as well

File: CONTRIBUTING.md (hunk 4 above)
Comment (SamDepardieu) [reply CONTRIBUTING.md]: Sounds right. I think issue conversations are the best way for us. Any thoughts about that?

**Comment by SamDepardieu** [comment]
@lizgzil @jdkram @nsorros This is ready for another round! 👌 ☕️

**Pull request merged**